h/static/scripts/directive/sidebar-tutorial.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+// @ngInject
+function controller($http, session, features) {
+  /*jshint validthis:true */
+  var vm = this;
+
+  vm.showSidebarTutorial = function () {
+    return (features.flagEnabled('sidebar_tutorial') &&
```

Comment:
I don't see there's any reason the client-side component needs to test the feature. Just don't add the field to the session server-side if the flag isn't enabled.